### ISSUE TITLE
Dashboard crashes after creating a project with a namespace default resource limit but no project resource limit

### ISSUE THREAD

**Issue by catherineluse** [rancher/dashboard#5154]
Steps to reproduce:

1. Create a project and go to the Resource Quotas tab.
2. Under Resource Type, select CPU Limit.
3. In the Namespace Default Limit field, enter 1000 mCPUs.
4. Click Save. the result is initially an error message `{"baseType":"error","code":"MissingRequired","fieldName":"name","message":"","status":422,"type":"error"}`
5. Continue filling out the form by entering a number in the Project Limit field. Then the app crashes with a white screen and the error message `Cannot read properties of undefined (reading 'limit')`

**Comment by neillsom** [comment]
Unable to reproduce, issue appears to be fixed, possibly from PR [#5000](https://github.com/rancher/dashboard/issues/5000)

**Comment by brudnak** [comment]
# :white_check_mark: PASSED

## Reproduction Environment

#### Not required.

<hr>
  
## Validation Environment

| Component                 | Version / Type                           |
| ------------------------- | ---------------------------------------- |
| Rancher version           | 2.6.5-rc2                                |
| Rancher commit link       | not applicable                           |
| Installation option       | helm high availability                   |
| RKE binary version used   | v1.3.8                                   |
| If Helm Chart k8s cluster | v1.22.7                                  |
| Cert Details              | let's encrypt                            |
| Docker version            | 20.10.7, build f0df350                   |
| Helm version              | v2.16.8-rancher1                         |
| Downstream cluster type   | rke1 linode                              |
| Downstream K8s version    | v1.22.8-rancher1-1                       |
| Logged in user role       | administrator & standard user            |
| Browser type              | google chrome                            |
| Browser version           | 100.0.4896.127 (official build) (x86_64) |

**ADDITIONAL SETUP**

- Helm commands for Rancher installation:

```shell
kubectl create namespace cattle-system
```

```shell
helm install cert-manager jetstack/cert-manager \
  --namespace cert-manager \
  --create-namespace \
  --version v1.7.1 \
  --set installCRDs=true
```

```shell
helm install rancher rancher-latest/rancher \
  --namespace cattle-system \
  --set hostname=$OMITTED \
  --set bootstrapPassword=admin \
  --set ingress.tls.source=letsEncrypt \
  --set letsEncrypt.email=$OMITTED \
  --set letsEncrypt.ingress.class=nginx \
  --version 2.6.5-rc2
```

## Validation steps

1. Create a project and go to the Resource Quotas tab.
2. Under Resource Type, select CPU Limit.
3. In the Namespace Default Limit field, enter 1000 mCPUs.
4. Click Save
5. Continue filling out the form by entering a number in the Project Limit field
   
## Additional Info

## RESULTS

#### :white_check_mark: Expected
Unable to reproduce, just as mentioned here https://github.com/rancher/dashboard/issues/5154#issuecomment-1092323633

#### :white_check_mark:Actual
Unable to reproduce, just as mentioned here https://github.com/rancher/dashboard/issues/5154#issuecomment-1092323633
